docker-bake.hcl

```diff
@@ -1,11 +1,18 @@
 target "aws" {
     target = "aws"
-    platforms = ["linux/amd64", "linux/arm64"]
+    platforms = ["linux/amd64"]
     args = {"BASE_IMAGE": "alpine:3.17"}
 }
 
 target "gcp" {
     target = "gcp"
-    platforms = ["linux/amd64", "linux/arm64"]
+    platforms = ["linux/amd64"]
     args = {"BASE_IMAGE": "gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine"}
-}
\ No newline at end of file
+}
+
+target "azure" {
+    target = "azure"
+    platforms = ["linux/amd64"]
+    args = {"BASE_IMAGE": "mcr.microsoft.com/azure-cli:2.48.1"}
```

Comment:
ideally, we'd use something like `mcr.microsoft.com/azure-cli:2` to have constant updates but apparently that's not a valid tag 😢 I wonder if we can risk `latest` tag

Reply:
let's keep it as is actually, we'll bump it every once it a while